understanding/20/captions-live.html

```diff
@@ -11,12 +11,12 @@ <h1>Understanding Captions (Live)</h1>
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
-		<dt>Objective</dt><dd>Live videos have captions</dd>
-		<dt>Author task</dt><dd>Provide synchronized text for audio content in real-time videos</dd>
-		
-		</dl>
-		
-		</section>
+         <dt>Goal</dt><dd>Live videos have captions.</dd>
+         <dt>What to do</dt><dd>Provide synchronized text for audio content in real-time videos.</dd>
+         <dt>Why it's important</dt><dd>People who are deaf or hard of hearing can understand real-time video content.</dd> 
```

Comment:
@GreggVan suggested
```suggestion
         <dt>Why it's important</dt><dd>People who are deaf or hard of hearing can understand real-time audio in video content.</dd> 
```

Reply:
If we wanted to add this, I think it would make more sense to put "audio" in front of "real-time" to read:
```suggestion
         <dt>Why it's important</dt><dd>People who are deaf or hard of hearing can understand audio in real-time video content.</dd> 
```